bindings/matrix-sdk-ffi/src/platform.rs

```diff
@@ -202,13 +203,28 @@ where
     }
 
     let file_layer = config.write_to_files.map(|c| {
+        let mut builder = RollingFileAppender::builder()
+            .rotation(Rotation::HOURLY)
+            .filename_prefix(&c.file_prefix);
+
+        if let Some(max_files) = c.max_files {
+            builder = builder.max_log_files(max_files as usize)
+        };
+        if let Some(file_suffix) = c.file_suffix {
+            builder = builder.filename_suffix(file_suffix)
+        }
+
+        let writer = builder
+            .build(&c.path)
+            .unwrap_or(tracing_appender::rolling::hourly(c.path, c.file_prefix));
```

Comment:
This seemed a reasonable fallback to me given
a) It does work fine, the extra parameters are just for refinement.
b) I assume that if we log the error and bail, its too early for that log to be written to a file anyway.

Reply:
Yeah, it'd be fine to `unwrap()` or `expect()` here actually, because... that's the same as the previous behavior; the `expect()` is plain hidden in the `tracing_appender` code. Can you do that please?

Reply:
Got it, updated 👍